reference/constraints/NoSuspiciousCharacters.rst

```diff
@@ -0,0 +1,157 @@
+NoSuspiciousCharacters
+======================
+
+.. versionadded:: 6.3
+
+    The ``NoSuspiciousCharacters`` constraint was introduced in Symfony 6.3.
+
+..
```

Comment:
This is a comment, and therefore not rendered on the website, am I correct @wouterj @javiereguiluz ?

Reply:
I hope so! I was following https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#block-quotes

> [Empty comments](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#empty-comments) may be used to explicitly terminate preceding constructs that would otherwise consume a block quote